www/src/pages/editor/editDedupSite/EditDedupMineralSite.tsx

```diff
@@ -143,9 +144,123 @@ export const EditDedupMineralSite = observer(({ dedupSite, commodity }: EditDedu
       message.success("Ungrouping was successful!");
     }
   };
-
+  let dynamicColumns = useMemo(() => {
+    const dynamicCols = [];
+    if (settingStore.hasGeologyInfo()) {
+      dynamicCols.push({
```

Comment:
Render geology info ( alternation, concentration process, etc.) as separate columns. no need to use descriptions.